website/docs/api/admin/events-api.md

```diff
@@ -2,11 +2,10 @@
 id: events
 title: /api/admin/events
 ---
+
```

Comment:
Would revert this file, unless there are any non-formatting changes?

Reply:
Done